src/main/java/io/confluent/idesidecar/restapi/util/OperatingSystemType.java

```diff
@@ -6,6 +6,7 @@ public enum OperatingSystemType {
   Windows,
   MacOS,
   Linux,
+  Solaris,
```

Comment:
added Solaris for more test granularity.

Reply:
Do we want `Solaris`, or `Unix` since Solaris is one of many Unixes?